ui/components/ui/color-indicator/color-indicator.scss

```diff
@@ -41,23 +41,23 @@ $sizes: (
 
   @each $variant, $color in design-system.$color-map {
     &--color-#{$variant} {
-      border-color: $color;
+      border-color: var($color);
       &#{$self}--partial-filled #{$self}__inner-circle {
-        background-color: $color;
+        background-color: var($color);
       }
       &#{$self}--filled {
-        background-color: $color;
+        background-color: var($color);
       }
       & #{$self}__icon {
-        color: #{utilities.choose-contrast-color($color)};
```

Comment:
Hmm. I see that we're no longer using this color contrast mixin. This was the sole use, so we may as well remove the mixin if we aren't using ti here.

This is an unfortunate change, as it means icons won't be assigned contrasting colors anymore. Does this always evaluate to black right now in practice? I guess we could solve this later using the theming system without the use of this mixin.

Reply:
Right; unfortunately SASS can't render or work with dynamic values from CSS variables, and the spots I noticed where rendering as black.  

IMO this change is worth it, as CSS variables will make theming miles more extensible moving forward.